chainservice/builder.go

```diff
@@ -539,8 +553,13 @@ func (builder *Builder) buildBlockSyncer() error {
 				retries = 4
 			}
 			var err error
+			opts := []blockchain.BlockValidationOption{}
+			if now := time.Now(); now.Before(blk.Timestamp()) ||
+				blk.Height()+cfg.Genesis.MinBlocksForBlobRequests > estimateHeight(blk, now.Sub(blk.Timestamp())) {
+				opts = append(opts, blockchain.ValidateSidecarOption())
+			}
```

Comment:
we could go the other way, that is simply compare the timestamp, no need to bother with the `estimateHeight` above
```
if blk.Timestamp() <= time.Now() - 20 days {
    opts = append(opts, blockchain.ValidateSidecarOption())
}
```